### ISSUE TITLE
ResNet50 model (available on github)

### ISSUE THREAD

**Issue by LikaOkudzhava** [LikaOkudzhava/BrainyScan#34]
1. Understand the parameters (the code and explanations are in the notebook)
2. Understand the difference to other models
3. Make a summary and discuss
4. Adjust the "Alzheimer model template" according to your model:

Input: Preprocessed training and testing data

-Define model architecture (construct keras model with pre-trained base):
- Followed by Flatten, Batch normalization and Dense layers
- Freeze base model
- model.build to finalize model structure

-Configure and compile the model:
- ModelCheckpoint to save the best model based on validation and earlystopping 
- specify optimizer, loss function and metrics

- Train the model
- Plot training/validation accuracy and loss.
- Evaluate on test data

Output: identify and apply parameters for pre-trained models adapted to MRI images (e.g., provided on kaggle),
implement on the subset of data and save each model to compare the performance
